src/js/utils/dom.js

```diff
@@ -56,6 +56,15 @@ function classRegExp(className) {
 }
 
 /**
+ * Whether the current DOM interface appears to be real.
+ *
+ * @return {Boolean}
+ */
+export function isReal() {
+  return document === window.document && typeof document.createElement === 'function';
```

Comment:
this would need a `typeof window !== 'undefined'` and `typeof document !== 'undefined'`, since otherwise they're a reference error in node.